assets/component-slider.css

```diff
@@ -88,34 +84,14 @@ slider-component {
   border-radius: 0.4rem;
 }
```

Comment:
On the article page, the same arrow has a different animation on hover: it grows and changes opacity (darker). I wonder if we should follow the same for both (on all paginations)

https://screenshot.click/24-43-a1is6-w64hg.mp4

Reply:
Yes! We just aligned Ken and I to follow the current pagination hover state! 👌